identity-actor/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "identity-actor"
+version = "0.5.0"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+homepage = "https://www.iota.org"
+keywords = ["iota", "tangle", "identity"]
+license = "Apache-2.0"
+readme = "../README.md"
+repository = "https://github.com/iotaledger/identity.rs"
+description = "A peer-to-peer identity communication framework"
+
+[dependencies]
+async-trait = { version = "0.1", default-features = false }
+dashmap = { version = "5.1", default-features = false }
+futures = { version = "0.3", default-features = false }
+identity-core = { version = "=0.5.0", path = "../identity-core", default-features = false }
+identity-iota = { version = "=0.5.0", path = "../identity-iota", default-features = false, features = ["async"] }
+identity-iota-core = { version = "=0.5.0", path = "../identity-iota-core", default-features = false }
+libp2p = { version = "0.45", default-features = false, features = ["tcp-tokio", "dns-tokio", "websocket", "request-response", "noise", "yamux"] }
+log = { version = "0.4", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+serde_json = { version = "1.0", default-features = false }
+thiserror = { version = "1.0", default-features = false }
+tokio = { version = "1.15", default-features = false, features = ["rt", "time"] }
+uuid = { version = "0.8", default-features = false, features = ["v4", "serde"] }
+
+[target.'cfg(all(target_arch = "wasm32", not(target_os = "wasi")))'.dependencies]
+libp2p = { version = "*", features = ["wasm-bindgen", "wasm-ext", "wasm-ext-websocket"] }
+uuid = { version = "*", features = ["wasm-bindgen"] }
+wasm-bindgen-futures = { version = "0.4.29", default-features = false }
```

Comment:
Since we decided to go with something like gRPC to support actors/agents for different programming languages I think this can be removed?